DashAI/back/dataloaders/classes/dashai_dataset.py

```diff
@@ -311,3 +311,46 @@ def save_dataset(datasetdict: DatasetDict, path: str) -> None:
             sort_keys=True,
             ensure_ascii=False,
         )
+
+
+@beartype
+def get_column_types(dataset_path: str) -> Dict[str, Dict]:
+    """Return the column with their respective types
+
+    Parameters
+    ----------
+    dataset_path : str
+        Path where the dataset is stored.
+
+    Returns
+    -------
+    Dict
+        Dict with the columns and types
+    """
+    dataset = load_dataset(dataset_path=dataset_path)
+    dataset_features = dataset["train"].features
+    column_types = {}
+    for column in dataset_features:
+        if dataset_features[column]._type == "Value":
+            column_types[column] = {
+                "type": "Value",
+                "dtype": dataset_features[column].dtype,
+            }
+        elif dataset_features[column]._type == "ClassLabel":
+            column_types[column] = {"type": "Classlabel", "dtype": ""}
```

Comment:
Just styles, add a final comma to the dict to make it multiline.
```suggestion
            column_types[column] = {"type": "Classlabel", "dtype": "",}
```